src/styles/vars.scss

```
@@ -0,0 +1,7 @@
+$accent_color: #00acdc;
+$gray_elements: #f3f3f3;
+$primary-color: #060b35;
+$secondary_color: #616070;
```

It is also better to set all naming in the same style.
```suggestion
$gray_elements: #f3f3f3;
$primary_color: #060b35;
$secondary_color: #616070;
```